package.json

```diff
@@ -38,7 +38,7 @@
   },
   "peerDependencies": {
     "@stripe/stripe-js": "^3",
-    "svelte": "^3 || ^4"
+    "svelte": "^3 || ^4 || ^5.0.0-next.120"
```

Comment:
looks like latest is currently [5.0.0-next.173](https://www.npmjs.com/package/svelte/v/5.0.0-next.173)

Reply:
Svelte 5 is now released.